server/src/models/index.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {commitsModel} from './commits';
```

Comment:
Actually, I followed the structure [here](https://softwareontheroad.com/ideal-nodejs-project-structure/).
I guess the word 'Model' is used because it is a database model, and also it works like a 'Model' in MVC.

Reply:
+1 for 'Storage', 'Model' sounds like it's more about the data representation, but here the main focus is that the data is persisted in and retrieved from database.

Reply:
@ifhuang @shengdac okay then I change to 'Storage'